plugins/audit.c

```diff
@@ -82,8 +84,13 @@ static rpmRC audit_tsm_post(rpmPlugin plugin, rpmts ts, int res)
 	    rasprintf(&eventTxt,
 		    "op=%s %s sw_type=rpm key_enforce=%u gpg_res=%u %s",
 		    op, nevra, enforce, verified, dir);
-	    audit_log_user_comm_message(auditFd, AUDIT_SOFTWARE_UPDATE,
-				    eventTxt, NULL, NULL, NULL, NULL, result);
+	    if (audit_log_user_comm_message(auditFd, AUDIT_SOFTWARE_UPDATE,
```

Comment:
Maybe this deserves a TODO comment, like the one in `removeSBITS()`, just so we can grep for these (in theory) in the future.

Reply:
Figured it's actually just as easy to do the right thing and make it a warning, just filter out the message you get when auditd isn't running, similarly to what we do in systemd_inhibit and dbus_announce.

Reply:
Oh neat, seems like the innocent nitpick turned into something more useful :smile:

Reply:
Yup. This is the kind of stuff that happens when you just want something quickly out of the way and instead of stopping to think for a second (because this is not the thing you're interested just now) you think you work around it somehow. And then later realize doing the right thing would've been at least as simple :facepalm: :smile:

Reply:
Yep, having someone else stop you in the tracks, if just for a second, unlocks your hidden potential sometimes :smile: